.changeset/gold-maps-cut.md

```diff
@@ -0,0 +1,100 @@
+---
+"@fluidframework/agent-scheduler": minor
```

Comment:
while technically true, i'd trim this remove example, tests, experimental, and cases where the change really shouldn't be impactful